services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(genres_ids: list[Genre] = None,
+               actors_ids: list[Actor] = None) -> Movie:
```

Comment:
If the argument can be `None` you must show it in typehint. Fix it everywhere.
```suggestion
def get_movies(genres_ids: list[Genre] | None = None,
               actors_ids: list[Actor] | None = None) -> Movie:
```